File: cc/007.cc
class Solution {
public:
    int Fibonacci(int n) {
        int f1 = 0, f2 = 1;
        while (n--)
        {
            f2 += f1;
            f1 = f2 - f1;
        }
        return f1;
    }
};

File: python/007.py
# -*- coding: utf-8 -*-
# /usr/bin/python3
# Creator: Hemajun
# Description: 007


class Solution:
    def Fibonacci(self, n):
        f1 = 0
        f2 = 1
        while n != 0:
            f2 += f1
            f1 = f2 - f1
            n -= 1
        return f1
